docs/writing-docs/docs-page.md

```diff
@@ -0,0 +1,190 @@
+Primary.parameters = { docs: { page: null } }
+```
+
+#### Component-level
+
+Override the `docs.page` [parameter](../writing-stories/parameters#component-parameters) in the default export of the story file.
+
+```js
+// Button.stories.js
+
+import { Button } from './Button';
+export default {
+  title: 'Storybook Examples/Button',
+  component: Button,
+  parameters: { 
+    docs: { 
+      page: null 
+    } 
+  },
+};
+```
+
+#### Global-level
+
+Override the `docs.page` [parameter](../writing-stories/parameters#global-parameters) in [`.storybook/preview.js`](../configure/overview#configure-story-rendering).
+
+```js
+// .storybook/preview.js
+
+export const parameters { docs: { page: null } });
+```
+
+### Remixing DocsPage using doc blocks
+
+Doc blocks are the basic building blocks of Storybook Docs. DocsPage composes them to provide a reasonable UI documentation experience out of the box. 
+
+If you want to make minor customizations to the default DocsPage but don’t want to write your own MDX you can remix DocsPage. That allows you to reorder, add, or omit doc blocks without losing Storybook’s automatic docgen capabilities. 
+
+Here's an example of rebuilding DocsPage for the Button component using doc blocks:
+
+```js
+// Button.stories.js
+
+import React from 'react';
+
+import {
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  Props,
+  Stories,
+} from '@storybook/addon-docs/blocks';
+
+import { Button } from './Button';
+
+export default {
+  title: 'Components/Button',
+  component: Button,
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle />
+          <Description />
+          <Primary />
+          <Props />
+          <Stories />
+        </>
+      ),
+    },
+  },
+};
+```
+
+Apply a similar technique to remix the DocsPage at the [story](#story-level), [component](#component-level), or [global](#global-level) level.
+
+In addition, you can interleave your own components to customize the auto-generated contents of the page, or pass in different options to the blocks to customize their appearance. Read more about [Doc Blocks](./docs-blocks).
+
+### Story file names
+<div>
+TODO ask tom/dom for a read on this. I went a bit of restructuring so that this page aligns with the mdx one in terms of wording/flow.
+</div>
+
+Unless you use a custom [webpack configuration](../configure/integration#extending-storybooks-webpack-config), all of your story files should have the suffix `*.stories.@(j|t)sx?`, e.g. "Badge.stories.js", "Badge.stories.tsx", etc. This tells Storybook and its docs preset to display the docs based on the file contents.
+
+### Inline stories vs. Iframe stories
+
+DocsPage displays all the stories of a component in one page. You have the option of rendering those stories inline or in an iframe. 
+
+By default, we render React and Vue stories inline. Stories from other supported frameworks will render in an `<iframe>`by default. 
```

Comment:
```suggestion
By default, we render React and Vue stories inline. Stories from other supported frameworks will render in an `<iframe>` by default. 
```

Reply:
added the space, marking it as resolved.